### ISSUE TITLE
"vagrant up" gives an error

### ISSUE THREAD

**Issue by leto** [fossasia/open-event-orga-server#118]
Goals:

- [ ] Our dependencies are installed in a dedicated place with virtualenv
- [ ] Our dependency files are owned by a dedicated user `oe`
- [ ] The above dependencies use a version of Python that we specify that is not the system Python
- [ ] At the end of a `vagrant up`, once can go to a URL and see the flask-admin interface running, without having to `vagrant ssh` into the VM and run additional commands

After lots of successful output, ```vagrant up``` gives:

```
==> default: Env variable for application development:
==> default:   DATABASE_URL=postgresql://open_event_user:start@localhost:15432/test
==> default:
==> default: Local command to access the database via psql:
==> default:   PGUSER=open_event_user PGPASSWORD=start psql -h localhost -p 15432 test
==> default: Traceback (most recent call last):
==> default:   File "<stdin>", line 1, in <module>
==> default: ImportError
==> default: :
==> default: cannot import name app
```

because there is no app.py

**Comment by leto** [comment]
Now I am getting 
```
==> default: Traceback (most recent call last):
==> default:   File "<stdin>", line 1, in <module>
==> default:   File "open_event/__init__.py", line 5, in <module>
==> default:
==> default: from flask import Flask
==> default: ImportError
==> default: :
==> default: No module named flask
```

**Comment by mbj36** [comment]
@leto 

I also got this 
Then i installed it by "sudo pip install flask"

And its working :)

**Comment by leto** [comment]
@mbj36 that is a temporary solution to work around this, yes. But this is a bug in our provision scripts, which should do this step

**Comment by leto** [comment]
I am working on this (and other related bugs) in the `gh_118` branch. We need to be installing our dependencies as a non-root user and not in the system-wide path using `virtualenv`.

**Comment by batraabhishek** [comment]
@leto I am getting same errors. Is this fixed in gh_118 branch.? :)

**Comment by creativepsyco** [comment]
@batraabhishek @leto I am trying to fix the vagrant related issues. You can find my changes under the `mohit_dev` branch, I think the repo requires quite a lot of cleanup especially from installation standpoint, I am having issues running the basic Flask app with Postgres. Guess need @rafalkowalski to take a look at it.

**Comment by creativepsyco** [comment]
Just found out that in [version 0.3 of Flask-Login](https://github.com/maxcountryman/flask-login/blob/master/CHANGES#L25-L26) they introduced a breaking change regarding model fields being properties instead of methods.

I think `mohit_dev` should have most issues resolved by tomorrow, some issues with requirements.txt and Database related issues were to blame.

I will clean it up, test it and retrofit the documentation for newbies and submit a PR.

Meanwhile you can check out [mohit_dev](https://github.com/fossasia/open-event-orga-server/tree/mohit_dev) for an updated version. It should run smoothly from start to finish, more instructions in the README.

Cheers

**Comment by rafalkowalski** [comment]
@creativepsyco I've fixed problems with version 0.3 could you check?

**Comment by leto** [comment]
This has been fixed, closing
